Sources/Testing/Events/Event.swift

```diff
@@ -98,6 +98,13 @@ public struct Event: Sendable {
     ///   - issue: The issue which was recorded.
     indirect case issueRecorded(_ issue: Issue)
 
+    /// An attachment was created.
+    ///
+    /// - Parameters:
+    ///   - attachment: The attachment that was created.
+    @_spi(Experimental)
+    indirect case valueAttached(_ attachment: Test.Attachment)
```

Comment:
Bikeshedding here, I realize a little prematurely (names don't have to be perfect right now or block this PR from landing) but… "**value**Attached" seems a bit odd. I'd suggest something like `attachhmentCreated` or `attachmentAdded`, for better symmetry with `issueRecorded` and `expectationChecked`.

Maybe the verb for this is just… record!? `attachmentRecorded` , for parity with `issueRecorded`?

Reply:
The verb in the actual function is `attach()` currently, so… `Attached`. `attachmentAttached` felt weird. :)

Let's plan to bikeshed this name before I put up a proposal doc.